package/firmware/linux-firmware/Makefile

```diff
@@ -8,12 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=linux-firmware
-PKG_VERSION:=20250613
+PKG_SOURCE_DATE:=2025-06-27
+PKG_SOURCE_VERSION:=f40eafe216833d083f4e5598b7f45e894c373ad1
 PKG_RELEASE:=1
 
-PKG_SOURCE_URL:=@KERNEL/linux/kernel/firmware
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_HASH:=d40743337ca8bf9d2587e220ab30dec6a07b52cc5cb4a4ff8d92be14391a7fde
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://gitlab.com/kernel-firmware/linux-firmware.git
+PKG_MIRROR_HASH:=003000331869f9a0188cc1115a5aa029c8505c5bb924ea3bbbde3db458782837
```

Comment:
I prefer the tar.xz files provided by the kernel team. 

Why do you prefer gitlab over git.kernel.org?

Reply:
I would still prefer using the existing tar.xz file.

Reply:
OK .. I will remove the 2nd commit from this PR.